contracts/src/v0.8/functions/dev/0_0_0/Functions.sol

```diff
@@ -13,7 +13,8 @@ library Functions {
 
   enum Location {
     Inline,
-    Remote
+    Remote,
+    DON
```

Comment:
I'm wondering if "DonHosted" or "DONHosted" is a better name?

"secrets are inline"
"secrets are remote"
"secrets are DON-hosted"